src/wallet/rpc/transactions.cpp

```diff
@@ -33,7 +33,9 @@ static void WalletTxToJSON(const CWallet& wallet, const CWalletTx& wtx, UniValue
         entry.pushKV("trusted", CachedTxIsTrusted(wallet, wtx));
     }
     uint256 hash = wtx.GetHash();
+    uint256 witness_hash = wtx.GetWitnessHash();
     entry.pushKV("txid", hash.GetHex());
+    entry.pushKV("wtxid", witness_hash.GetHex());
```

Comment:
In fe65dab3dbef51e49b484e155ffbf7fd01ea2e8e "wallet: add wtxid in WalletTxToJSON"

nit: `witness_hash` is not used anywhere else, so it does not need to be a separate variable

```suggestion
    entry.pushKV("wtxid", wtx.GetWitnessHash().GetHex());
```

Reply:
Done!